test/colony-network.js

```diff
@@ -385,11 +386,45 @@ contract("ColonyNetwork", accounts => {
       assert.equal(resolvedAddress, colonyAddress);
 
       // Check reverse lookup
-      const lookedUpAddress = await colonyNetwork.lookupUsername(colonyAddress);
-      assert.equal(lookedUpAddress, "test.colony.joincolony.eth");
+      const lookedUpENSDomain = await colonyNetwork.lookupUsername(colonyAddress);
+      assert.equal(lookedUpENSDomain, "test.colony.joincolony.eth");
 
       // Can't register two labels for a colony
       await checkErrorRevert(colony.registerColonyLabel(colonyName2, { from: accounts[0] }), "colony-already-labeled");
     });
+
+    it("should be able to register same name for user and a colony, and reverse lookup still work", async () => {
+      // Register user
+      await colonyNetwork.registerUserLabel("test", orbitDBAddress, { from: accounts[1] });
+
+      // Set up colony
+      const token = await Token.new(...TOKEN_ARGS);
+      const { logs } = await colonyNetwork.createColony(token.address);
+      const { colonyAddress } = logs[0].args;
+      const colony = await Colony.at(colonyAddress);
+      // Register colony
+      // Owner can register label for colony
+      await colony.registerColonyLabel("test", { from: accounts[0] });
+
+      // Check reverse lookup for colony
+      const lookedUpENSDomainColony = await colonyNetwork.lookupUsername(colonyAddress);
+      assert.equal(lookedUpENSDomainColony, "test.colony.joincolony.eth");
+
+      // Check reverse lookup
+      const lookedUpENSDomainUser = await colonyNetwork.lookupUsername(accounts[1]);
+      assert.equal(lookedUpENSDomainUser, "test.user.joincolony.eth");
+    });
+
+    it("should return a blank address if looking up an address with no Colony-based ENS name", async () => {
+      const lookedUpENSDomain = await colonyNetwork.lookupUsername(accounts[2]);
+      assert.equal(lookedUpENSDomain, "");
+    });
+
+    it("should respond correctly to queries regarding ENS interfaces it supports", async () => {
+      let response = await colonyNetwork.supportsInterface("0x01ffc9a7"); // supports 'supportsInterface(bytes4)'
+      assert(response);
```

Comment:
Can we use `assert.isTrue` as this if error simply produces a non-specific error `AssertionError: Unspecified AssertionError`